cms/static/cms/js/modules/cms.modal.js

```diff
@@ -265,8 +298,56 @@
                 this.ui.modal.focus();
             },
 
-            _hide: function (duration) {
+            /**
+             * Closes the current instance.
+             *
+             * @method close
+             */
+            close: function close() {
+                var that = this;
+
+                // handle remove option when plugin is new
+                if (CMS._newPlugin) {
```

Comment:
i think this can be renamed into something more appropriate, like `CMS._isCreatingNewPlugin`. is it really directly on CMS namespace? i thought it was on modal itself